src/nft/components/bag/BagHeader.tsx

```diff
@@ -47,6 +43,26 @@ const IconWrapper = styled.button`
     },
   }) => `opacity ${duration.medium} ${timing.ease}`};
 `
+const NumberDot = styled.div`
+  align-items: center;
+  background-color: ${({ theme }) => theme.accentActive};
+  border-radius: 50%;
```

Comment:
border radius should be 100px according to Figma, it makes a visual difference and looks closer to the design.